examples/README.md

```diff
@@ -0,0 +1,148 @@
+
+**Certificate bundles** are added as a file-mounted configmap at `/etc/pki/tls/custom-certs/ca-bundle.crt`.
+This is a nonstandard location, so it is necessary to also add environment variables that instruct various software to reference this bundle during operation.
+
+Reference:
+* https://github.com/opendatahub-io/kubeflow/blob/eacf63cdaed4db766a6503aa413e388e1d2721ef/components/odh-notebook-controller/controllers/notebook_webhook.go#L598
+* https://github.com/opendatahub-io/kubeflow/blob/eacf63cdaed4db766a6503aa413e388e1d2721ef/components/odh-notebook-controller/controllers/notebook_webhook.go#L601-L607
+
+**Pipeline runtime configuration** is obtained from a Data Science Pipeline Application (DSPA) CR.
+The DSPA CR is first located in the same project where the workbench is being started, a secret with the connection data is created, and then this secret is mounted.
+The secret is mounted under `/opt/app-root/runtimes/`.
+
+Reference: https://github.com/opendatahub-io/kubeflow/blob/eacf63cdaed4db766a6503aa413e388e1d2721ef/components/odh-notebook-controller/controllers/notebook_dspa_secret.go#L42C28-L42C50
+
+IMPORTANT: the `setup-elyra.sh` script in this repo relies on this location.
+
+**Runtime images** are processed very similarly to the DSPA configuration.
+First, image stream resources are examined, and then a configmap is created, and mounted to every newly started workbench.
+The mount location is under `/opt/app-root/pipeline-runtimes/`.
+
+Reference: https://github.com/opendatahub-io/kubeflow/blob/eacf63cdaed4db766a6503aa413e388e1d2721ef/components/odh-notebook-controller/controllers/notebook_runtime.go#L25C19-L25C51
+
+IMPORTANT: the `setup-elyra.sh` script in this repo again relies on this location.
+
+### Build
+
+```shell
+podman build -f examples/jupyterlab-with-elyra/Dockerfile -t quay.io/your-username/jupyterlab-with-elyra:latest .
+podman push quay.io/your-username/jupyterlab-with-elyra:latest
+```
+
+### Deploy
+
+Open the `Settings > Workbench images` page in OpenDataHub Dashboard.
+Click on the `Import new image` button and add the image you have just pushed.
+The `Image location` field should be set to `quay.io/your-username/jupyterlab-with-elyra:latest`, or wherever the image is pushed and available for the cluster to pull.
+Values of other fields do not matter for functionality, but they let you keep better track of previously imported images.
+
+There is a special ODH Dashboard feature that alerts you when you are using a workbench image that lists the `elyra` instead of `odh-elyra` package.
+This code will have to be updated when `elyra` also gains support for Argo Pipelines, but for now it does the job.
+
+Reference: https://github.com/opendatahub-io/odh-dashboard/blob/2ced77737a1b1fc24b94acac41245da8b29468a4/frontend/src/concepts/pipelines/elyra/utils.ts#L152-L162
+
+## Image Streams
```

Comment:
remove this, already covered in https://github.com/opendatahub-io/notebooks/blob/main/docs/workbench-imagestreams.md

Reply:
removed